atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/ConflictHandler.java

```diff
@@ -23,7 +23,7 @@
     IGNORE_ALL(false, false, false, false),
 
     /**
-     * If two transactions concurrently write to the same cell the one that commits later will
+     * If two transactions concurrently write to the same row the one that commits later will
```

Comment:
This is wrong; Atlas will always retry on the cell cahnging not the row

Reply:
Basically, there are two things at play here:

1. The granularity of your locking: do you lock on the row or the cell?
2. The types of conflicts you will see.

Your comment affects 2, but write write and write write cell are identical from that perspective.

Reply:
not sure I understand the distinction.

To make this concrete, say I have:

```
row("id", uuid)
column("value_a", int)
column("value_b", int)

Transaction X
put(id=1, value_a=4)

Transaction Y
put(id=1, value_b=10)
```

What is the behaviour of WRITE_WRITE and WRITE_WRITE_CELL? Do neither of them conflict?

Reply:
WRITE_WRITE: X and Y block on each other via a row lock but don't/can't conflict (conflict checks are always at the cell level)

WRITE_WRITE_CELL: X and Y can run concurrently and can't conflict